store/metadata.go

```diff
@@ -201,6 +201,10 @@ func (ms *MetadataStore) getLinkedPeersInfo() (map[string]bool, []metadata.Conta
 			} else {
 				for _, aService := range linkedServices {
 					for _, aContainer := range aService.Containers {
+						// Skip containers with --net=host
+						if aContainer.PrimaryIp == ms.info.hostsMap[aContainer.HostUUID].AgentIP {
```

Comment:
there might be a slight chance that host is removed, and no longer present in hostsMap, while instance is still in metadata (as instance cleanup from the removed host can happen a bit later). Would make sense to add a guard against it.